docs/src/main/asciidoc/sqs.adoc

```diff
@@ -1231,6 +1231,8 @@ public interface MessagingMessageConverter<S> {
 
 The default header-based type mapping can be configured to use a different header name by using the `setPayloadTypeHeader` method.
 
+Also is possible to not send `TypeHeader` by using the `doNotSendTypeHeader` method.
```

Comment:
We can use a similar approach to the javadocs here, something like:

"It is also possible not include payload type information in the header by using..."